### PR TITLE
[android] Resources and Java classes cleaning

### PR DESCRIPTION
**Pull request by arnaudvergnet**
Hello, here I come with yet another large PR. This one is not UI/UX related so it should be easier ;)

I removed unused Java classes as well as any unused resource, be it a layout, drawable, values... **About 450 files were removed**, resulting in smaller APK (haven't actually checked, maybe the build already striped unused stuff), better DX, and faster Android Studio.

Most of those changes should be checked at build time. The app builds, runs and I could not find any regression. There should not be any visible change, if you find some please notify me! This is mainly a developer facing change.

To find unused files/resources, I used the "Find usages" and "Find in files" features of Android studio.

### CODE REVIEW

**Review by biodranik — APPROVED** [review]
Fantastic work, Arnaud! Thank you very much for the cleanup!

Can you please rebase it?

File: android/res/values/colors.xml [hunk 1]
```diff
@@ -147,75 +123,16 @@
...
-  <color name="turquoise">#D0F6FF</color>
-  <color name="bookmarks_subs_title">#c60b2b1b</color>
-  <color name="bookmarks_subs_title_night">#c6ffffff</color>
-  <color name="bookmarks_subs_subtitle">#c6152b1c</color>
-  <color name="bookmarks_subs_subtitle_night">#c6ffffff</color>
-  <color name="sale_view_bg">#A335F4</color>
-  <color name="sightseeing_subs_sale_view_bg">#E54E35</color>
-  <color name="bg_sightseeing_subs">#F0FCFF</color>
-  <color name="bg_placeholder">#E2E2E2</color>
+  <color name="bg_editor_light_pressed">#3D000000</color>>
```
Comment (strump) [inline android/res/values/colors.xml:128]: Extra `>` in the end. Please remove

Comment (biodranik) [reply android/res/values/colors.xml:128]: Wow, great catch! Thanks! I'm pushing the fix into the master.

Comment (arnaudvergnet) [reply android/res/values/colors.xml:128]: Damn how is that even compiling?

**Pull request merged**